src/devices/cpu/t11/t11.h

```diff
@@ -13,11 +13,6 @@ enum
 	T11_R0=1, T11_R1, T11_R2, T11_R3, T11_R4, T11_R5, T11_SP, T11_PC, T11_PSW
 };
 
-#define T11_IRQ0        0      /* IRQ0 */
-#define T11_IRQ1        1      /* IRQ1 */
-#define T11_IRQ2        2      /* IRQ2 */
-#define T11_IRQ3        3      /* IRQ3 */
```

Comment:
These input line constants are used in at least src/mame/dec/pdp11.cpp and possibly other places.  The code currently doesn’t compile.  If you give them a new home (e.g. as `static inline constexpr` inside `t11_device`), please do a full build and update everything that uses them.

Reply:
I'll update that to use existing enum (t11_device::CP0_LINE)